tvm/src/codegen/hlsc/build_hlsc.cc

```diff
@@ -34,29 +34,55 @@ TVM_REGISTER_API("codegen.build_vhls_csim")
 #endif
 
 template<class CodeGen>
-std::string BuildHLSC(Array<LoweredFunc> funcs) {
+std::string BuildHLSC(
+    Array<LoweredFunc> funcs, int output_mode) {
   CodeAnalysMerlinC ca;
   CodeGen cg;
   for (LoweredFunc f : funcs) {
     // 1st pass: Analyze AST and collect necessary information
     ca.AddFunction(f);
     str2tupleMap<std::string, Type> map_arg_type;
     map_arg_type = ca.Finish();
+    if (output_mode == 2) {
```

Comment:
We should use an enum in the future. Otherwise it's hard to tell what is the "2" mode.

Reply:
please fix it now. we must avoid hardcoding constants (including strings and paths) in our program.